### PR TITLE
unison-fsmonitor support

### PR DESCRIPTION
**Pull request by popsikle**
Just a quick change to add the unison-fsmonitor support added in the last version.

### CODE REVIEW

**Comment by leighmcculloch** [comment]
@popsikle does fsmonitor function properly on Mac? I've read of issues with it and Mac OS X in the past.

**Comment by popsikle** [comment]
This runs within the container, and yes it functions.  I can't seem to get OSX to pick up changes and sync them back to the container, but the reverse (which this PR enabled) works.

**Comment by smyth64** [comment]
Gonna try this!

File: unison-run.sh [hunk 1]
```diff
@@ -1,4 +1,7 @@
 #!/bin/sh
 export HOME=/root
 cd /unison
+if [ -f /bin/unison-$UNISON_VERSION-fsmonitor ]; then
+  ln -s -f /bin/unison-$UNISON_VERSION-fsmonitor /bin/unison-fsmonitor
+fi
```
Comment (leighmcculloch) [inline unison-run.sh:6]: I'm inclined to do the linking in the install script. Why do it here?

File: Dockerfile [hunk 2]
```diff
@@ -1,4 +1,4 @@
-FROM phusion/baseimage:0.9.16
+FROM phusion/baseimage
```
Comment (leighmcculloch) [inline Dockerfile:1]: @popsikle is this necessary for using fsmonitor? I'd like to remove it if not, and do it in a separate commit if it'd be better not to pin to the version.

**Comment by leighmcculloch** [comment]
@popsikle Sorry for the delay! I've dropped two questions on the files. Let me know what you think and then I'll merge it and cut a new version.

File: unison-run.sh (hunk 1 above)
Comment (popsikle) [inline unison-run.sh:6]: To be honest, I dont remember at all.  Its been a while heh.  I think it was because at one point I had multiple versions  installed and wanted to use the right fsmonitor for the version that was starting, but I dont think thats the case anymore.

File: Dockerfile (hunk 2 above)
Comment (popsikle) [inline Dockerfile:1]: Probably not needed anymore.  There was an issue with the docker repo when I was testing this and pulling down that version, as well as some ca-cert issues, but you can probably tack it to the latest if you wanted to.

**Comment by leighmcculloch** [comment]
Closing as this will be done in #11.
